packages/staking/src/features/drawer/StakePoolConfirmation.tsx

```diff
@@ -84,7 +98,7 @@ const StakePoolConfirmationBody = ({
         >
           <ItemStatRenderer
             img={stakePool.displayData.logo}
-            text={stakePool.displayData.name || '-'}
+            text={<EllipsizedPoolName {...{ stakePool }} />}
```

Comment:
```suggestion
            text={<EllipsizedPoolName stakePool="stakePool" />}
```
Alternatively, allow for passing just a pool name.